app/_layouts/landing-page.html

```diff
@@ -58,7 +58,7 @@ <h5 class="platform-description">
       </section>
 
       <h3>
-        Get started with Kong Konnect
+        Get started with Kong Konnect Cloud
```

Comment:
I think in most places within the Konnect documentation, we can just refer to "Kong Konnect".  However, since this is the main landing page with ambigous Kong Enterprise "Konnect" vs "Kong Konnect Cloud", I thought it'd be better to be clear.